Assets/VRTK/Scripts/Controls/3D/VRTK_Knob.cs

```diff
@@ -163,8 +196,23 @@ private KnobDirection DetectDirection()
 
         private float CalculateValue()
         {
+            if (!subDirectionFound)
+            {
+                float angleX = Mathf.Abs(transform.localRotation.eulerAngles.x - initialLocalRotation.x) % 90;
+                float angleY = Mathf.Abs(transform.localRotation.eulerAngles.y - initialLocalRotation.y) % 90;
+                float angleZ = Mathf.Abs(transform.localRotation.eulerAngles.z - initialLocalRotation.z) % 90;
+                angleX = (Mathf.RoundToInt(angleX) > 89 && Mathf.RoundToInt(angleX) < 91) ? 0 : angleX;
```

Comment:
What does this do? You're already % 90 above so shouldn't it always be 0 <= angle <= 90?